src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -106,7 +106,7 @@ CHIP_ERROR ExampleOperationalCredentialsIssuer::Initialize(PersistentStorageDele
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateNOCChainAfterValidation(NodeId nodeId, FabricId fabricId,
+CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateNOCChainAfterValidation(NodeId nodeId, FabricId fabricId, CASEAuthTag cat,
```

Comment:
Why only one CAT?

Reply:
Fixed that. Changed that to the list of CATs (CATValues).